app/models/form/sales/pages/nationality1.rb

```diff
@@ -7,15 +7,12 @@ def initialize(id, hsh, subsection)
         "privacynotice" => 1,
       },
       {
-        "noint" => 1,
+        "buyer_not_interviewed?" => true,
       },
     ]
   end
 
   def questions
-    @questions ||= [
-      Form::Sales::Questions::Nationality1.new(nil, nil, self),
-      Form::Sales::Questions::OtherNationality1.new(nil, nil, self),
-    ]
+    @questions ||= [Form::Sales::Questions::Nationality1.new(nil, nil, self)]
   end
```

Comment:
Are we meant to be removing it for both years?

Reply:
yes. clarified with Miguel